docs/development/docs-contribute.md

```diff
@@ -0,0 +1,153 @@
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+Druid is a [community-led project](https://druid.apache.org/community/), and we are delighted to receive contributions of anything from minor fixes to docs to big new features.
+
+Druid docs contributors:
+
+* Improve existing content
+* Create new content
+
+## Getting started
+
+Druid docs contributors can open an issue about documentation, or contribute a change with a pull request (PR).
+
+The open source Druid docs are located here:
+https://druid.apache.org/docs/latest/design/index.html
+
+
+If you need to update a Druid doc, locate and update the doc in the Druid repo, following the instructions below.
+
+## Druid repo branches
+
+The Druid team works on the `master` branch and then branches for a release, such as 25.0
+
+See [`CONTRIBUTING.md`](https://github.com/apache/incubator-druid/blob/master/CONTRIBUTING.md) for instructions on contributing to Apache Druid.
+
+## Before you begin
+
+Before you can contribute to the Druid docs for the first time, you must complete the following steps:
+1. Fork the [Druid repo](https://github.com/apache/druid). Your fork will be the `origin` remote.
+2. Clone your fork: 
+   
+   ```bash
+   git clone git@github.com:GITHUB_USERNAME/druid.git
+   ```
+3. In the directory where you cloned your fork, set up `apache/druid`  as your your remote `upstream` repo:
+   
+   ```bash
+   git remote add upstream https://github.com/apache/druid.git
+   ```
+4. Confirm that your fork shows up as the origin repo and `apache/druid` shows up as the upstream repo:   
+
+   ```bash
+   git remote -v
+   ```
+
+5. Verify that you have your email configured for GitHub:
+
+   ```bash
+   git config user.email
+   ```
+   
+   If you need to set your email, see the [GitHub instructions](https://docs.github.com/en/github-ae@latest/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git).
+
+5. Install Docusaurus so that you can build the site locally. Run either `npm install` or `yarn install` in the `website` directory.
+   
+## Contributing
+
+Before you contribute, it's a good idea to make sure your local branch of `master` and the upstream Apache branch are up-to-date and in sync. This can help you avoid merge conflicts. Run the following commands on your fork's `master` branch:
```

Comment:
```suggestion
Before you contribute, make sure your local branch of `master` and the upstream Apache branch are up-to-date and in sync. This can help you avoid merge conflicts. Run the following commands on your fork's `master` branch:
```